components/Kebab.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import { useState } from 'react';
+import { Dropdown } from './Dropdown';
+
+function Kebab() {
+  const [isKebabOpen, setIsKebabOpen] = useState<boolean>(false);
+
+  const handleKebabToggle = () => {
+    setIsKebabOpen((prev) => !prev);
+  };
+
+  return (
+    <div className='kebab'>
+      <Image
+        src='/images/ic_kebab.svg'
+        alt='kebab'
+        height={24}
+        width={24}
+        onClick={handleKebabToggle}
+      />
```

Comment:
```suggestion
  <button onClick={handleKebabToggle}>
  <Image
    src='/images/ic_kebab.svg'
    alt='메뉴 열기'
    height={24}
    width={24}
  />
</button>
```
이렇게 버튼요소 감싸 주는것이 접근성 측면에서 조금 좋습니다!